spec/unit/face/config_spec.rb

```diff
@@ -23,6 +23,19 @@ module PuppetFaceSpecs
     OUTPUT
   end
 
+  it "prints environment_timeout=unlimited correctly" do
+    Puppet[:environment_timeout] = "unlimited"
+
+    expect { subject.print("environment_timeout") }.to have_printed('unlimited')
+  end
+
+  it "prints arrays correctly" do
+    skip "Still doesn't print arrays like they would appear in config"
```

Comment:
I think we want to use `pending` here so that the test will fail if the code is later changed to print arrays correctly (so we know which tests to go update)